charts/seqr/values.yaml

```diff
@@ -179,6 +174,56 @@ redis:
     enabled: false
   fullnameOverride: "seqr-redis"
 
+postgresql:
+  enabled: false
+  architecture: standalone
+  auth:
+    username: postgres
+    existingSecret: *postgresSecretName
+    secretKeys:
+      # Keyword used to find the admin password inside the existingSecret
+      adminPasswordKey: password
+  fullnameOverride: seqr-postgresql
+  image:
+    tag: '12.19.0-debian-12-r9'
+  postgresqlDataDir: /seqr/postgresql-data
+  primary:
+    persistence:
+      existingClaim: |-
+        {{ include "lib.pvc-name" . }}
+      mountPath: /seqr
+    # We ran into race condition issues with the liveness probe killing
+    # the postgres init and leaving partial state in the data dir.
+    # Some details here: https://github.com/bitnami/charts/issues/14683
+    startupProbe:
+      enabled: true
+
+    # This is a reduced initContainers intended to replace the default
+    # provided with volumePermissions.enabled set to True.  The default
+    # uses the /data subdirectory and ignores postgresqlDataDir
+    # leading to an incorrectly permissioned data directory.
+    # See here: https://github.com/bitnami/charts/blob/aa038e9953df2ba74d481cb7ba1cccb3dc32c5fc/bitnami/postgresql/templates/primary/statefulset.yaml#L117-L157
+    initContainers:
```

Comment:
There was additional futzing to make the volume work correctly in a sane way.  By default, the PGDATA would be mounted to `/seqr/data`, and I thought it was nicer to have the directory include `postgresql` in some way.  `/seqr/postgresql-data` fits the pattern with the rest of our directory names.